python/ray/runtime_env/runtime_env.py

```diff
@@ -266,6 +270,9 @@ class MyClass:
         config: config for runtime environment. Either
             a dict or a RuntimeEnvConfig. Field: (1) setup_timeout_seconds, the
             timeout of runtime environment creation,  timeout is in seconds.
+        image_uri: URI to a container image. The Ray worker process runs
+            in a container with this image. You can't use this parameter with
+            other fields of runtime_env.
```

Comment:
you can use it with `config` and `env_vars`

Reply:
updated